sagemaker-huggingface/text-summarizer/huggingface-custom-text-summarizer.ipynb

```diff
@@ -0,0 +1,2626 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    s3_prefix = "samsum-dataset"
</pre><p>Preferred to use a DEMO prefix, such as <code>DEMO-samsum-dataset</code> ﻿, so it's easier for customers to recognize later what this data was used for.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>